refarch-apigateway/apigateway/src/main/resources/application-test.yml

```diff
@@ -0,0 +1,63 @@
+wiremock:
+  server:
+    port: 0
+    https-port: -1
+
+spring:
+  
+  # muenchen.de realm is used by default so that the SpringContext can be initalized upon running the tests. Change to your own dev realm if available
+  realm: muenchen.de
+
+  # spring cloud gateway config
+  cloud:
+    gateway:
+      actuator:
+        verbose:
+          enabled: false
+      globalcors:
+        corsConfigurations:
+          # The cors configuration to allow frontend developers cross origin request via this api gateway
+          '[/**]':
+            allowedOrigins:
+              - "http://localhost:8081"
+              - "http://127.0.0.1:8081"
+            allowedMethods: "*"
+            allowedHeaders: "*"
+            allowCredentials: true
+            maxAge: 3600
+      routes:
+        - id: sso
+          uri: https://ssotest.muenchen.de/
```

Comment:
Should be changed to a local instance? Maybe also Mock instance with wiremock is possible?

Reply:
added variable and we like i said in the other comment, maybe testcontainers could help